.travis.yml

```diff
@@ -27,23 +28,20 @@ install:
 # ./dist/setup/setup here instead of cabal-install to avoid breakage when the
 # build config format changed.
 script:
- - cabal sandbox init
+ # We depend on parsec nowadays, which isn't distributed with GHC <8.0
```

Comment:
May want to make this conditional to make sure that Cabal can be built with stock GHC 8.0.